lib/sikuli/common.sikuli/common.py

```diff
@@ -385,7 +385,7 @@ def _screenshot_and_time(self, width, height,
         current_time = time.time()
 
         # Print log message for recording
-        self.common.system_print(action_name)
+        General().system_print(action_name)
```

Comment:
due to we did not use `self`, it seems like we can change `_screenshot_and_time()` from instance method to static method.